### PR TITLE
Enabling Flight Masters

### LINKED ISSUE #17778: Flight Paths not working
**Description**:  Flight Paths not working

**Current behaviour**: You can click on a Gryphon Master and try to fly to a known flight location... and it never mounts you to fly there.

**Expected behaviour**:  Mount on a flight path to go to the selected location

**Steps to reproduce the problem**:

Click on a Gryphon Master and select a flight location.

**Branch(es)**: 6.x

**TC hash/commit**:  e6ddb9f2220b

**TDB version**:  TDB 6.04

**Operating system**:  

Server:  Ubuntu 16.04.1 LTS
Client OS: Windows 10 and MacOS 10.11.6
Client Version:  7.0.3.22423

### PR DESCRIPTION
**Pull request by lorac2k14**
**Changes proposed:**
-  Enabling the opcode to utilize the flight master handler

**Target branch(es):** 6.x

**Issues addressed:** Closes #17778

**Tests performed:** (Does it build, tested in-game, etc)
It builds with no warnings and has been tested by both campaigns
No crashes no issues and runs as it should 

**Known issues and TODO list:**
- Flightpaths need fine tuned for the new world 
  I need to research how the flight paths work so maybe I can help address this 
- Seems to still lack a lot of functionality especially in new content 
-  Closed for now

### CODE REVIEW

File: src/server/game/Server/Protocol/Opcodes.cpp [hunk 1]
```diff
@@ -128,7 +128,7 @@ void OpcodeTable::Initialize()
     DEFINE_HANDLER(CMSG_ACCEPT_LEVEL_GRANT,                                 STATUS_LOGGEDIN,  PROCESS_THREADUNSAFE, &WorldSession::HandleAcceptGrantLevel);
     DEFINE_HANDLER(CMSG_ACCEPT_TRADE,                                       STATUS_LOGGEDIN,  PROCESS_THREADUNSAFE, &WorldSession::HandleAcceptTradeOpcode);
     DEFINE_HANDLER(CMSG_ACCEPT_WARGAME_INVITE,                              STATUS_UNHANDLED, PROCESS_INPLACE,      &WorldSession::Handle_NULL);
-    DEFINE_HANDLER(CMSG_ACTIVATE_TAXI,                                      STATUS_UNHANDLED, PROCESS_THREADSAFE,   &WorldSession::HandleActivateTaxiOpcode);
+    DEFINE_HANDLER(CMSG_ACTIVATE_TAXI,                                      STATUS_LOGGEDIN,  PROCESS_THREADSAFE,   &WorldSession::HandleActivateTaxiOpcode);
```
Comment (MitchesD) [inline src/server/game/Server/Protocol/Opcodes.cpp]: you need to confirm that structure did not change

Comment (Shauren) [inline src/server/game/Server/Protocol/Opcodes.cpp]: I updated the structure, didn't enable opcode

Comment (lorac2k14) [inline src/server/game/Server/Protocol/Opcodes.cpp]: I know this is a newb question but how do I find the new fields so I can look at the changes and try to provide handlers for the new fields or did you already address the missing fields in commit 59cb6740ffe0b9b7cb2a22ff474971fffb570974

**Comment by lorac2k14** [comment]
Closing PR since it is not a fix and its a fixhack and not getting any feedback as to how to address what makes this a hack

**Comment by Traesh** [comment]
I'll look at this later this week, but if you still want to work on this PR, two new fields has been added : 

            uint32 GroundMountID = 0;
            uint32 FlyingMountID = 0;

It's a new functionnality allowing player to travel on one of his mount, maybe only a favorite one, need to be determined. It should be handled before packet could be activated.

You can gather data by sniffs, trying to get on travel with one of your mounts (seem like it's a random chance to me)

**Comment by Shauren** [comment]
@Traesh yes, client selects one random favorite mount for these fields but the catch here is that only one flight master uses this - the one in Dalaran on Broken Isles (probably there is a flag for this that needs to be researched)

**Pull request closed without merging**